pulp_smash/tests/pulp3/pulpcore/api_v3/test_crud_distributions.py

```diff
@@ -78,3 +79,63 @@ def test_05_delete(self):
         self.client.delete(self.distribution['_href'])
         with self.assertRaises(HTTPError):
             self.client.get(self.distribution['_href'])
+
+
+class DistributionBasePathTestCase(unittest.TestCase):
+    """Test possible values for ``base_path`` on a distribution.
+
+    This test targets the following issues:
+
+    * `Pulp #3142 <https://pulp.plan.io/issues/3142>`_
```

Comment:
This is a typo. This should link to issue 3412.

Reply:
It is. Fixing right now.